src/pykeen/datasets/analysis.py

```diff
@@ -616,6 +616,11 @@ def relation_cardinality_classification(
         .. math ::
             1 = \sum_{type} conf(relation, type)
 
+    .. note ::
+       These relation types are also mentioned in TransH [wang2014]_. However, the paper does not provide any details on
```

Comment:
are these part of the TransH model? if not, maybe it's just better to not mention the model name and just make the citation, since this kind of work is not model-specific

Reply:
No they are not part of. The authors argue that their model is particularly well-suited to deal with these more complex relation types.